data-intake/planning-data-load/main.py

```diff
@@ -0,0 +1,351 @@
+from typing import Dict
+
+import requests
+import pandas as pd
+import os
+from dotenv import load_dotenv
+from pandas import DataFrame
+
+from planSchema import *
+
+load_dotenv()
+API_KEY = os.getenv('VISIER_API')
+VISIER_HOST = os.getenv('VISIER_HOST')
+BASE_URI = "v1/planning"
+
+PLAN_ID = os.getenv('PLAN_ID')
+SCENARIO_ID = os.getenv('SCENARIO_ID')
+
+METHOD = os.getenv('METHOD')
+CALCULATION = os.getenv('CALCULATION')
+
+SOURCE_FILENAME = os.getenv('SOURCE_FILENAME')
+MAPPED_DATA_LOAD_FILENAME = os.getenv('MAPPED_DATA_LOAD_FILENAME')
+MAPPED_ADD_REMOVE_FILENAME = os.getenv('MAPPED_ADD_REMOVE_FILENAME')
+
+MISSING_ROW_RCIS = ['RCIP972016', 'RCIP980022', 'RCIP980021']
+
+
+# Authenticates using the Visier API
+def authenticate():
+    print("Authenticating with Visier...")
+    url = f"{VISIER_HOST}/v1/admin/visierSecureToken?vanityname={os.getenv('VISIER_VANITY')}"
+    payload = {
+        'username': os.getenv('VISIER_USERNAME'),
+        'password': os.getenv('VISIER_PASSWORD')
+    }
+    headers = {
+        'Content-Type': 'application/x-www-form-urlencoded'
+    }
+
+    response = requests.request("POST", url, headers=headers, data=payload)
+    if response.status_code != 200:
+        print(response.json())
+        print("Error encountered. Exiting from script...")
+        exit()
+    else:
+        return response.text
+
+
+# Uses the Planning Data Load API to load data into a plan.
+# Returns the response and any missing row indices.
+def load_data_into_visier(filename, method) -> DataLoadResult:
+    print(f"Loading file: {filename} into Visier using calculation={CALCULATION} and method={method}...")
+    security_token = authenticate()
+    url = f"{VISIER_HOST}/{BASE_URI}/data/plans/{PLAN_ID}/scenarios/{SCENARIO_ID}?calculation={CALCULATION}&method={method}"
```

Comment:
All sample applications in this repo is using python-sdk. 
https://github.com/visier/python-sdk

For this endpoint the package should be `visier_api_data_in`.
and the API name is `PlanningDataLoadApi`
https://github.com/visier/python-sdk/blob/main/src/visier_api_data_in/visier_api_data_in/api/planning_data_load_api.py